src/aat/AATFeatureMap.js

```diff
@@ -478,14 +478,14 @@ for (let ot in OTMapping) {
 // in the form of {featureType:{featureSetting:true}}
 export function mapOTToAAT(features) {
   let res = {};
-  for (let k = 0; k < features.length; k++) {
+  for (let k in features) {
     let r;
-    if (r = OTMapping[features[k]]) {
+    if (r = OTMapping[k]) {
```

Comment:
actually based on the fact that you're treating the new `k` as content out of the `features` array, you definitely want `of` in the above loop:
```
for (let i in ['a', 'b', 'c']) { console.log(i); } // yields 0, 1, 2
for (let i of ['a', 'b', 'c']) { console.log(i); } // yields a, b, c
```